attribute_set_completeness/models/attribute_set.py

```diff
@@ -16,11 +16,41 @@ class AttributeSet(models.Model):
         auto_join=True,
     )
 
+    # Add a button to let the user choose between automatic completion
+    # rate vs manual input
+    completeness_mode = fields.Boolean(default=True)
```

Comment:
I would add an help text to guide the user and would provide a better name for a boolean like 'is_automatic_rate' or something like that